desc/objectives/objective_funs.py

```diff
@@ -902,16 +919,18 @@ def compute(self, *args, **kwargs):
 
     def compute_unscaled(self, *args, **kwargs):
         """Compute the raw value of the objective."""
-        return jnp.atleast_1d(self.compute(*args, **kwargs))
+        return jnp.atleast_1d(
+            self._loss_function(jnp.atleast_1d(self.compute(*args, **kwargs)))
```

Comment:
Does this need the extra `jnp.atleast_1d` call here? `computed_scaled` and `compute_scaled_error` don't have this.

Reply:
I put it in case loss function returned something that was not a 1D ndarray (like jnp.mean() which returns a 0D array)